common-compose.yml

```diff
@@ -36,7 +36,7 @@ services:
       - PATH_PREFIX=/api/v1
       - CROMWELL_CREDENTIALS=/app/jobs/config.json
       - CAPABILITIES_CONFIG=/app/jobs/capabilities_config.json
-      - CROMWELL_URL=${CROMWELL_URL:-https://cromwell.mint-dev.broadinstitute.org/api/workflows/v1}
```

Comment:
Since there are more engineers work on Job Manager now, it doesn't make sense to keep this default value here (mint team-specific) anymore.